Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/orient_polygon_soup.h

```diff
@@ -108,8 +110,8 @@ struct Polygon_soup_orienter
   {
     typedef std::pair<V_ID,P_ID> VID_and_PID;
     if ( is_edge_marked(src,tgt,marked_edges) ) return VID_and_PID(src,300612);
-    Edge_map_iterator em_it=edges.find(V_ID_pair(tgt, src));
-    if ( em_it==edges.end() ) return VID_and_PID(src,300612);// the vertex is on the border
+    Edge_map_iterator em_it=edges[tgt].find(src);
+    if ( em_it==edges[tgt].end() ) return VID_and_PID(src,300612);// the vertex is on the border
```

Comment:
If you want to use magic number, use sound ones, like `-1`, but not arbitrary integers!

Reply:
The change does not introduce this number it was there before.